tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "@ownclouders/tsconfig",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "allowJs": true,
+    "paths": {
+      "@/*": ["./src/*"]
+    },
+    "outDir": "./dist",
+    "module": "ESNext",
+    "moduleResolution": "node"
+  },
+  "vueCompilerOptions": {
+    "extensions": [".vue"],
+    "target": 2
+  }
+}
```

Comment:
Could extend the config from the monorepo

Reply:
It is already extending it, I'm just adding couple of things needed that are not set in that config.